### PR TITLE
Stack implementation

### PR DESCRIPTION
**Pull request by kshitij10496**
Added both the header as well as the `.c` file.
Tested on the system.

### CODE REVIEW

File: src/stack.h [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+
+            PUSH(S, 1);
+            PUSH(S, 2);
+            PUSH(S, 3);
+            POP(S);
+
+            PRINT_STACK(S);
+            return 0;
+        }
+		[Out]: 1 2
```
Comment (OrkoHunter) [inline src/stack.h:26]: The Python touch :smiley:

File: src/stack.h [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+// Function to add an element to the Stack
+void PUSH(struct Stack*, int);
+
+// Function to remove an element from the Stack 
+int POP(struct Stack*);
+
+// Function to print the Stack
+void PRINT_STACK(struct Stack*);
+
+#endif
```
Comment (OrkoHunter) [inline src/stack.h]: Always add a new line at the end of a file.

File: src/stack.c [hunk 3]
```diff
@@ -0,0 +1,63 @@
+#include "stack.h"
+
+struct sNode* __init__snode(int value) {
```
Comment (OrkoHunter) [inline src/stack.c:4]: Add a comment about the local function.

File: src/stack.h (hunk 1 above)
Comment (kshitij10496) [inline src/stack.h:26]: Yeah !

**Pull request merged**